crates/wadm-types/src/bindings.rs

```diff
@@ -5,9 +5,10 @@ use crate::{
     },
     CapabilityProperties, Component, ComponentProperties, ConfigDefinition, ConfigProperty,
     LinkProperty, Manifest, Metadata, Policy, Properties, SecretProperty, SecretSourceProperty,
-    Specification, Spread, SpreadScalerProperty, TargetConfig, Trait, TraitProperty,
+    SharedApplicationComponentProperties, Specification, Spread, SpreadScalerProperty,
+    TargetConfig, Trait, TraitProperty,
 };
-use wasmcloud::wadm;
+use wasmcloud::wadm::{self};
```

Comment:
This is likely going to trigger a clippy lint and should just be `wasmcloud::wadm`